frontend/src/components/editor/actions/useNotebookActions.tsx

```diff
@@ -362,61 +343,20 @@ export function useNotebookActions() {
         }
       },
     },
+
     {
-      icon: <ZapOffIcon size={14} strokeWidth={1.5} />,
-      label: "Disable all cells",
-      hidden: !hasEnabledCells || kioskMode,
-      handle: async () => {
-        const notebook = getNotebook();
-        const ids = enabledCellIds(notebook);
-        const newConfigs = Objects.fromEntries(
-          ids.map((cellId) => [cellId, { disabled: true }]),
-        );
-        // send to BE
-        await saveCellConfig({ configs: newConfigs });
-        // update on FE
-        for (const cellId of ids) {
-          updateCellConfig({ cellId, config: { disabled: true } });
-        }
-      },
-    },
-    {
-      icon: <EyeOffIcon size={14} strokeWidth={1.5} />,
-      label: "Hide all markdown code",
-      handle: hideAllMarkdownCode,
-    },
-    {
-      icon: <ChevronRightCircleIcon size={14} strokeWidth={1.5} />,
-      label: "Collapse all sections",
-      hotkey: "global.collapseAllSections",
-      handle: collapseAllCells,
-    },
-    {
-      icon: <ChevronDownCircleIcon size={14} strokeWidth={1.5} />,
-      label: "Expand all sections",
-      hotkey: "global.expandAllSections",
-      handle: expandAllCells,
-    },
-    {
+      divider: true,
       icon: <DiamondPlusIcon size={14} strokeWidth={1.5} />,
       label: "Add setup cell",
       handle: () => {
         addSetupCellIfDoesntExist({});
       },
     },
     {
-      icon: <XCircleIcon size={14} strokeWidth={1.5} />,
-      label: "Clear all outputs",
+      icon: <DatabaseIcon size={14} strokeWidth={1.5} />,
+      label: "Add database connection",
       handle: () => {
-        clearAllCellOutputs();
-      },
-    },
-    {
-      icon: <FastForwardIcon size={14} strokeWidth={1.5} />,
-      label: "Re-run all cells",
```

Comment:
Niche use case covered in most cases by restart kernel; still available in the command palette